app/components/file_component.html.erb

```diff
@@ -48,7 +48,7 @@
           </tr>
         </thead>
         <tbody>
-          <%= render Embed::File::DirRowComponent.new(viewer: viewer, dir: viewer.purl_object.hierarchical_contents)%>
+          <%= render Embed::File::DirRowComponent.new(viewer: viewer, dir: viewer.purl_object.hierarchical_contents, level_step: 0)%>
```

Comment:
Could we just revert the rest of this and send `level: -1` here?

Reply:
that's smarter. lol